packages/@o3r/schematics/src/tasks/eslint/index.ts

```diff
@@ -1,22 +1,59 @@
 import { TaskConfiguration, TaskConfigurationGenerator } from '@angular-devkit/schematics';
 import { NodePackageName, NodePackageTaskOptions } from '@angular-devkit/schematics/tasks/package-manager/options';
-import * as path from 'node:path';
 import { getPackageManager } from '../../utility/package-manager-runner';
 
+/**
+ * Linter options
+ */
+export interface LinterOptions {
+  /**
+   * Indicates if the linter process should succeed even if there are lint errors remaining
+   *
+   * @default true
+   * @deprecated will be removed in v10, please use `continueOnError`
+   */
+  force?: boolean;
+  /**
+   * Indicates if the linter process should succeed even if there are lint errors remaining
+   *
+   * @default true
+   */
+  continueOnError?: boolean;
+  /**
+   * If enabled, only errors are reported (--quiet option of ESLint CLI)
+   *
+   * @default true
+   */
+  hideWarnings?: boolean;
```

Comment:
maybe you can call it `showWarning` and `exitOnError` to default to `false` :D

Reply:
As discussed 
Deprecated `hideWarnings` in favor to `showWarning` is not that important.
Change `continueOnError` to `exitOnError` is not a good idea because we loose the fact that we will exit successfully even if there are errors.